src/azure-openapi-validator/tests/composite-azure-test.ts

```diff
@@ -0,0 +1,21 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import { Message } from '../../jsonrpc/types';
+import { suite, test, slow, timeout, skip, only } from "mocha-typescript";
+import { safeLoad } from "js-yaml";
+import { AutoRestPluginHost } from "../../jsonrpc/plugin-host";
+import { run } from "../../azure-openapi-validator";
+import { AssertValidationRuleCount, CollectTestMessagesFromValidator, ReadFileAsString } from './utilities/tests-helper';
+
+
+@suite class CompositeAzureTest {
+  @test @timeout(120000) async "description should not be parameter name"() {
+    const file = 'src/azure-openapi-validator/tests/resources/DescriptionMustNotBeNodeName.json';
```

Comment:
the ones where we expect violations are at https://github.com/Azure/autorest/tree/master/src/modeler/AutoRest.Swagger.Tests/Resource/Swagger/Validation, the positive cases are under "positive" folder, I wonder if we should have a folder for each (positive and negative) or want to follow the same thing, throwing all negative ones at the root and adding a folder for positive ones.